Assets/Scripts/Map/Pathfinder.cs

```diff
@@ -5,6 +5,7 @@
 public class Pathfinder: IPlayerSubscriber
 {
     Player m_Player;
+    List<Enemy> m_EnemyList;
```

Comment:
Momentan am decis ca Pathfinder-ul, cel care are matricea cu distanta la player, o sa tina cont de tile-urile blocate de inamici.
Nu suna ok.
De revizuit poate in viitor.